tiles-request-api/src/main/java/org/apache/tiles/request/locale/URLApplicationResource.java

```diff
@@ -21,20 +21,18 @@
 
 package org.apache.tiles.request.locale;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.net.JarURLConnection;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.net.URL;
-import java.net.URLConnection;
-import java.util.Locale;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.*;
+import java.net.*;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Set;
```

Comment:
Trivial:
 the import order was correct how it was. the project puts java.* imports first, and doesn't wildcard imports.